includes/orphaned_objects.inc

```diff
@@ -230,6 +320,30 @@ function islandora_delete_orphaned_objects_create_batch(array $pids) {
   return $batch;
 }
 
+/**
+ * Solr query to check for deceased parents.
+ *
+ */
+function islandora_identify_dead_parents($parent) {
```

Comment:
Not to be a stick in the mud, but I think we might want to find a better term for the parents of orphaned objects than "dead" or "deceased". I understand where you're coming from but I worry this might be in bad taste. Perhaps "missing" or "deleted" or even "nonexistent" would be better.

Reply:
Gosh man. I feel naming is such a complex thing. I know where you are coming from but this is a function name, just code invisible to anyone else.. not something exposed to the public. 'Orphan' is also a bad word. I feel kinda complex to find good names for anything that is related to parent ship and children... anyhow. @bondjimbond I'm changing this function anyway, I'm proposing some larger refactor in the code I'm writing, but will leave all the naming to you! But then also, my idea is kinda ambitious. Maybe you want a simpler one. As Islandora 7 faith is also to become "missing"  or past tense 😃

Reply:
Point taken! I was basing it on the existing language in the SPARQL code, but that stuff was probably my fault as well. I'm changing to "missing".

@DiegoPino I'm addressing all of Bryan's review before I start delving into your major refactor ideas.